### ISSUE TITLE
Shuttles do not move backwards or forwards

### ISSUE THREAD

**Issue by devkoks** [space-wizards/space-station-14#23888]
## Description
The shuttles do not move forward/backward, but move right/left and turn, but this only happens when running the server on Debian 12 OS.

**Reproduction**
Build and run on Debian 12 OS. And try to fly forward and backward on the shuttle.

**Screenshots**
A screenshot is not necessary in this situation.

**Additional context**
When testing on a local machine (Windows OS), everything works correctly, and previously worked on CentOS 7 OS

**Comment by VasilisThePikachu** [comment]
Closing due to the discussion that happened in #hosting in the discord server
https://discord.com/channels/310555209753690112/1010977110933901323/1194802292382244944

TLDR: This is most likely not an operating system bug. If it is please attach a debugger to the server and get to the bottom of this
